src/shared/services/common-data-service/common-data-service.ts

```diff
@@ -18,14 +18,21 @@ export interface LovRefDataModel {
     from?: string;
   }
 
+  export interface LovRefDataByServiceModel {
+    list_of_values: LovRefDataModel[];
+  }
+
 @Injectable()
 export class CommonDataService {
 
     constructor(private readonly http: HttpClient,
                 private readonly appconfig: AbstractAppConfig) {}
 
-    public getRefData(): Observable<LovRefDataModel[]> {
-            const url = '/assets/getCaseReasons.json';
-            return this.http.get<LovRefDataModel[]>(url)
+    public getRefData(): Observable<LovRefDataByServiceModel> {
+            const url = this.appconfig.getCaseLinkingReasonCommonDataAPIUrl();
```

Comment:
if this is getRDCommonDataAPIUrl, url can be concatenated with:
${getRDCommonDataAPIUrl}+lov/categories/CaseLinkingReasonCode